aries_vcx/src/indy/primitives/credential_schema.rs

```diff
@@ -25,16 +23,90 @@ pub struct Schema {
     pub name: String,
     pub source_id: String,
     #[serde(default)]
+    submitter_did: String,
+    #[serde(default)]
     pub state: PublicEntityStateType,
+    #[serde(default)]
```

Comment:
lets at least add a short comment like 
`field added in 0.45.0, we use #[serde(default)] for backwards compatibility`